packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -7,22 +7,32 @@ const providerName = require('../../lib/constants').ProviderName;
 const columnify = require('columnify');
 const { formUserAgentParam } = require('./user-agent');
 
+const CFN_MAX_CONCURRENT_REQUEST = 15;
+const CFN_POLL_TIME = 7000; // 7sec
+
+const CFN_SUCCESS_STATUS = [
+  'UPDATE_COMPLETE',
+  'CREATE_COMPLETE',
+  'DELETE_COMPLETE',
+  'DELETE_SKIPPED',
+];
```

Comment:
Maybe rename it TERMINAL_STATUS and add all the statuses that do not need further checking?